desktop_version/src/KeyPoll.cpp

```diff
@@ -93,7 +93,7 @@ void KeyPoll::Poll()
 
 			if (textentrymode)
 			{
-				if (evt.key.keysym.sym == SDLK_BACKSPACE)
+				if (evt.key.keysym.sym == SDLK_BACKSPACE && keybuffer.size() > 0)
```

Comment:
Please pardon me, I am just curious. Would it be better to use here **keybuffer.empty** instead of **keybuffer.size**?